### DISCUSSION TITLE
[Study schedule] 01/28 9PM(CST)

### DISCUSSION THREAD

**Discussion by swiri021** [OpenKBC/multiple_sclerosis_proj#111] [Announcements / Schedule]
New member has been joined the team !, please welcome new member!!

**Comment by kicheolkim** [comment]
**Current meeting agenda**
- Introduce each other, and multiple sclerosis
- Introduce Hyunmin's works (drug challenge)

**Next meeting agenda**
- Kicheol: personal capstone project for edX Data Science course, CIS vs Healthy, identify early biomarker using machine learning
- Nicole: paper presentation (DNA damage repair genes in Zebra fish)
- Next meeting will be held in 2-3 weeks (maybe 2/18 or 2/25 Friday (Saturday in Korea)?)

**Idea**
- identify target from gene expression dataset => design PROTAC
- pathway-based target discovery?
- isoform level analysis using EBseq (in current MS dataset)
- gene expression profile in MS (or other diseases): collect public gene expression dataset and _**visualization**_ | _**combined analysis or meta-analysis**_ | **_pathway/network/connection_ between cells/tissues** => this could be our specialized analysis pipeline for gene expression dataset
